klasses/models.py

```diff
@@ -82,6 +85,34 @@ def payment_deadline(self):
         """
         return self.installment_set.aggregate(payment_deadline=models.Max('deadline'))['payment_deadline']
 
+    @property
+    def next_installment(self):
+        """
+        Get the next installment
+        """
+        return self.installment_set.exclude(
```

Comment:
I talked to Ferdi and he told me that the deadline should be mandatory, so I will change the model and remove this code.